imports/server/addPuzzle.ts

```diff
@@ -149,11 +149,9 @@ export default async function addPuzzle({
 
   // Run any puzzle-creation hooks, like creating a default document
   // attachment or announcing the puzzle to Slack.
-  Meteor.defer(
-    Meteor.bindEnvironment(() => {
-      void GlobalHooks.runPuzzleCreatedHooks(puzzleId);
-    }),
-  );
+  Meteor.defer(() => {
+    void GlobalHooks.runPuzzleCreatedHooks(puzzleId);
```

Comment:
Huh, did I miss this `void` without a catch or a comment?

Reply:
Yeah I guess so, although the hook runners catch and log exceptions so I think it's fairly safe